pkg/api/pod/util.go

```diff
@@ -256,6 +256,31 @@ func DropDisabledAlphaFields(podSpec *api.PodSpec) {
 	}
 
 	DropDisabledVolumeDevicesAlphaFields(podSpec)
+
+	DropDisabledRunAsGroupField(podSpec)
+}
+
+// DropDisabledRunAsGroupField removes disabled fields from PodSpec related
+// to RunAsGroup
+func DropDisabledRunAsGroupField(podSpec *api.PodSpec) {
```

Comment:
need a corresponding function for dropping the RunAsGroup fields from PodSecurityPolicy objects, called from the podsecuritypolicy.strategy PrepareForCreate/PrepareForUpdate methods

Reply:
this is fixed now